### PR TITLE
LPD-41726 buildLang

### PR DESCRIPTION
**Pull request by kevenleone**

### CODE REVIEW

**Comment by kevenleone** [comment]
ci:test:sf

File: modules/apps/marketplace/marketplace-settings-web/src/main/resources/META-INF/resources/init.jsp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+<%@ page import="com.liferay.portal.kernel.util.HashMapBuilder" %><%@
+page import="com.liferay.portal.kernel.util.PortalUtil" %>
+
+<liferay-frontend:defineObjects />
+
+<liferay-theme:defineObjects />
+
+<liferay-util:html-top>
+	<aui:link href='<%= PortalUtil.getStaticResourceURL(request, PortalUtil.getPathProxy() + application.getContextPath() + "/css/main.css") %>' rel="stylesheet" type="text/css" />
```
Comment (CrescenzoRegaLF) [inline modules/apps/marketplace/marketplace-settings-web/src/main/resources/META-INF/resources/init.jsp:24]: You are creating a link to this css file but it doesn't actually exist in your module

Comment (kevenleone) [reply modules/apps/marketplace/marketplace-settings-web/src/main/resources/META-INF/resources/init.jsp:24]: Good point @CrescenzoRegaLF 

I can remove this

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:stop

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward:force

**Pull request closed without merging**